server/api/src/rsPhyPathReg.cpp

```diff
@@ -482,7 +508,7 @@ namespace {
             fileStatInp_t fileStatInp;
             memset( &fileStatInp, 0, sizeof( fileStatInp ) );
 
-            // Issue #3658 - This section removes trailing slashes from the 
+            // Issue #3658 - This section removes trailing slashes from the
             // directory path in the server when the path is already in the catalog.
             //
             // TODO:  This is a localized fix that addresses any trailing slashes
```

Comment:
did we find a better/existing way to do this work?

Reply:
Yes, a suggestion was made here: https://github.com/irods/irods/pull/5154#discussion_r491771767

I can fix this if desired, just not clear how to test other than running the suite again

Reply:
Nope, no worries - just looked familiar.   Carry on.

Reply:
Please make a new issue so we don't lose the suggestion